commands/dev-deploy.js

```diff
@@ -69,8 +69,15 @@ async function createDevAccountIfNeeded({ near, keyStore, networkId, init }) {
             }
         }
     }
+    let accountId;
+    if (typeof masterAccount === 'undefined' && helperUrl) {
```

Comment:
Is the code that provides a helpful message for prod in this PR, or was this some other PR?

Reply:
This isn't that code, but I won't have time for that in this PR turns out. Let's just consider that another small issue in my opinion, and consider this PR complete.